virtualBridge.py

```diff
@@ -69,7 +69,7 @@ def setup_dhcp_entry(self, cfg: NodeConfig) -> None:
         cmd = f"virsh net-update default add ip-dhcp-host \"{host_xml}\" --live --config"
         self.hostconn.run_or_die(cmd)
 
-    def _ensure_started(self, bridge_xml: str, api_port: str) -> None:
+    def _ensure_started(self, bridge_xml: str, api_port: Optional[str]) -> None:
```

Comment:
Pardon my confusion, but not sure what setting this as optional does?

API port should never be none, since we won't access this function if this is the case: https://github.com/bn222/cluster-deployment-automation/blob/main/clusterHost.py#L93